### PR TITLE
[TASK] Fix #687: Allow other extensions to add actions to the news administration module

### LINKED ISSUE #687: Allow other extensions to add actions to the news administration module
Sometimes it is necessary to add more news related activities for editors. Making a separate Backend module for that is not feasible. It makes sense to add functions to the existing news administration module instead. This can be achieved if the news administration controller has a way to extend its action menu by other extensions via signals. Than extensions will be able to add their actions to that menu and integrate extra functionality to the main news administration module.

### PR DESCRIPTION
**Pull request by dmitryd**
Sometimes it is necessary to add more news related activities for editors. Making a separate Backend module for that is not feasible. It makes sense to add functions to the existing news administration module instead. This can be achieved if the news administration controller has a way to extend its action menu by other extensions via signals. Than extensions will be able to add their actions to that menu and integrate extra functionality to the main news administration module.

### CODE REVIEW

File: Classes/Controller/AdministrationController.php [hunk 1]
```diff
@@ -177,12 +178,35 @@ protected function createMenu()
...
         $this->view->getModuleTemplate()->getDocHeaderComponent()->getMenuRegistry()->addMenu($menu);
         if (is_array($this->pageInformation)) {
             $this->view->getModuleTemplate()->getDocHeaderComponent()->setMetaInformation($this->pageInformation);
         }
     }
 
+    /**
+     * Extends menu selextor with items from 3rd party extensions.
+     *
+     * @param \TYPO3\CMS\Backend\Template\Components\Menu\Menu $menu
```
Comment (georgringer) [inline Classes/Controller/AdministrationController.php]: Fqn is not needed here

**Comment by georgringer** [comment]
@dmitryd do you want to take the comments into account and push a new change?

**Comment by dmitryd** [comment]
@georgringer 

Yes, I still have it on my list. But I work for 10 hours per day recently and did not have any free time. I expect to work for the same customer, who needed this change, very soon. Than I will be able to update this PR.

**Comment by dmitryd** [comment]
Working on it.

**Comment by dmitryd** [comment]
@georgringer Both fixed but integration for v9 fails and I am not sure what to do next :(

**Comment by georgringer** [comment]
@dmitryd do you wanna fix the last thing added by my comment

**Comment by dmitryd** [comment]
https://github.com/georgringer/news/pull/688/commits/3a54bc402bad84e8616fe35d3f7f163912c95c05

I am not getting notifications from github, so I react slowly to messages here. Sorry about that.

**Pull request merged**